src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
+
+|Keys|Action|
+|---|---|
+|`Ctrl` + `K` or `⌘` + `K`| access command center|
+|`ENTER`|execute command center command|
+
+# Shortcuts
+
+Only in the Appwrite SDK console use these keyboard shortcuts to access Appwrite's SDK console.  Using the first letter from the call to action and the resource/product/service/page, Appwrite's keyboard shortcuts follow this pattern.  For example, the shortcut `G` then `S` goes to storage, but if you want to go to settings, the shortcut is `G` then `E` because `G` then `S` is already taken, so the next letter of settings is used.
+
+|Shortcut|Action|
+|----|----|
+|Go to Auth|`G` then `A`|
+|Go to Storage|`G` then `S`|
+|Go to Databases|`G` then `D`|
+|Go to Functions|`G` then `F`|
+|Go to Projects|`G` then `P`|
+|Go to Overview|`G` then `O`|
+|Go to Settings|`G` then `E`|
+|Go to Messaging|`G` then `M`|
+
+# The Appwriter
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+In the Appwriter, Appwrite icons are used from the Appwrite Console and Appwrite's Docs to make it easier to memorize Appwrite keyboard shortcuts.  For example, the shortcut for going to databases is `G` then `D`, and the icon on the keyboard matches the icon in Appwrite Databases.
```

Comment:
Missing context for the reader, what is the Appwriter? why did we create it? I would also add some more technical info on the keyboard. Dylan or Jade can help with that. Then I would say how you can get it, soon on the Appwrite store during our init events (with link) or follow us on social media or Discord (links) to watch for giveaways we do from time to time.